usaspending_api/disaster/tests/integration/test_disaster_agency_spending.py

```diff
@@ -23,7 +23,7 @@ def test_basic_success(client, disaster_account_data, elasticsearch_award_index,
             "award_count": None,
             "obligation": 11000000.0,
             "outlay": 11.0,
-            "total_budgetary_resources": 2398472389.78,
+            "total_budgetary_resources": 23984723890.78,
```

Comment:
Why does this multiply by ten? I don't see `23984723890.78` anywhere in our fixtures, and the fact that this is a perfect multiple makes me suspicious that something is being multi-counted.

Reply:
Look in the rest of the PR for updated fixtures